app/services/medical_copays/vbs/service.rb

```diff
@@ -117,6 +117,10 @@ def get_pdf_statement_by_id(statement_id)
       end
 
       def get_copay_response
+        if Flipper.enabled?(:debts_copay_logging)
+          Rails.logger.info("MedicalCopays::VBS::Service#get_copay_response request data: #{@user.uuid}")
```

Comment:
Is logging the `uuid` necessary here?

Reply:
It would be super helpful to keep this. We need the Vet's EDIPI number to help troubleshoot an issue VBS is seeing in their system. Since we can't log EDIPI, I'll be able to get that info for VBS using the UUID.

Maybe you know of a better way?

Reply:
I know it's logged elsewhere. nah, I don't know a better way 😄